Hymns/Display/DisplayHymnViewModel.swift

```diff
@@ -91,20 +91,29 @@ class DisplayHymnViewModel: ObservableObject {
                     self.chordsUrl = chordsPath.flatMap({ path -> URL? in
                         HymnalNet.url(path: path)
                     })
+                    if self.chordsUrl != nil {
+                        WebView.preload(url: self.chordsUrl!)
```

Comment:
In your stack overflow link (https://stackoverflow.com/questions/48839180/preload-multiple-local-webviews), the WebViewPreloader there isn't static but is an object. If you follow that pattern, it'll be much easier to test/assert that you are preloading the correct thing here. Cuz currently, there's no way to assert this in your unit test

Reply:
Okay, I moved to a class. I tried to get away from using a static var but the problem is then the WebView representable struct complains about changing state in the view. So I have left it using a static var but made the functions associated with the class.